library/dd-sdk-android-session-replay/apiSurface

```diff
@@ -7,7 +7,9 @@ data class com.datadog.android.sessionreplay.SessionReplayConfiguration
     fun useSite(com.datadog.android.DatadogSite): Builder
     fun useCustomEndpoint(String): Builder
     fun setPrivacy(SessionReplayPrivacy): Builder
+    fun sessionReplaySampleRate(Float): Builder
```

Comment:
we must use `setSessionReplaySampleRate` for the consistency with changes done in sdk v2 (and drop `session replay` from the name if possible, we are already in the Session Replay configuration context).

```suggestion
    fun setSessionReplaySampleRate(Float): Builder
```

Reply:
@jamiemilstein are you ok with the name change here ? I know you specifically requested that this should be the same as in browser.

Reply:
hmm im confused why we need the `set`? I worry this differs from all other patterns so far. but if it's because of V2 changes and it needs to be that way i'm okay with it